KeywordSearch/src/org/sleuthkit/autopsy/keywordsearch/AccountsText.java

```diff
@@ -227,11 +227,17 @@ synchronized private void loadPageInfo() throws IllegalStateException, TskCoreEx
             }
 
             //add both the canonical form and the form in the text as accountNumbers to highlight.
-            this.accountNumbers.add(artifact.getAttribute(TSK_KEYWORD).getValueString());
-            this.accountNumbers.add(artifact.getAttribute(TSK_CARD_NUMBER).getValueString());
+            BlackboardAttribute attribute = artifact.getAttribute(TSK_KEYWORD);
+            if (attribute != null) {
+                this.accountNumbers.add(attribute.getValueString());
+            }
+            attribute = artifact.getAttribute(TSK_CARD_NUMBER);
+            if (attribute != null) {
+                this.accountNumbers.add(attribute.getValueString());
+            }
```

Comment:
Probably.  I ran into this (via NPE) when testing my fix.  I seemed like this code:
{code}
 if (attrType.equals(BlackboardAttribute.ATTRIBUTE_TYPE.TSK_CARD_NUMBER)) {
                attributeMap.put(new BlackboardAttribute.Type(BlackboardAttribute.ATTRIBUTE_TYPE.TSK_KEYWORD),
                        new BlackboardAttribute(BlackboardAttribute.ATTRIBUTE_TYPE.TSK_KEYWORD, MODULE_NAME, value));
                value = CharMatcher.anyOf(" -").removeFrom(value);
            }
           
{code}

from RegexQuery.java wasn't executing, and so there was no keyword attribute.   Checking both for null seemed like a reasonable safety precaution and,  to be honest, I didn't investigate further since the highlighting was working based on the card number attribute, and I forgot about this anomaly.